smartmin/views.py

```diff
@@ -141,10 +141,6 @@ def lookup_obj_attribute(self, obj, field):
         # next up is the object itself
         obj_field = getattr(obj, curr_field, None)
 
-        # if it is callable, do so
-        if obj_field and getattr(obj_field, '__call__', None):
```

Comment:
Isn't this used in say table views, where you can specify a column that is actually a method on the object? Or is this different? The former is used all over the place and is pretty useful. (it's also something the django-admin does as I recall so has precedent)